### PR TITLE
qa/cephadm: do not start upgrades from Octopus

### PR DESCRIPTION
**Pull request by adk3798**
now that we officially have v18 in main the upgrade
will fail if upgrading from Octopus (v15) to main
because they are now considered 3 major releases apart

Fixes: https://tracker.ceph.com/issues/57695

Signed-off-by: Adam King <adking@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/suites/orch/cephadm/mgr-nfs-upgrade/1-bootstrap/octopus.yaml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-tasks:
-- cephadm:
-    roleless: true
-    image: quay.io/ceph/ceph:v15
```
Comment (mgfritch) [inline qa/suites/orch/cephadm/mgr-nfs-upgrade/1-bootstrap/octopus.yaml:4]: also upgrade from `ceph:v16`  (16.2.0)?

**Review by mgfritch — APPROVED** [review]

**Comment by adk3798** [comment]
https://pulpito.ceph.com/adking-2022-10-05_00:18:12-orch:cephadm-wip-adk-testing-2022-10-04-1806-distro-default-smithi/

lots of
```
Command failed on smithi088 with status 17: "sudo /home/ubuntu/cephtest/cephadm --image quay.io/ceph/ceph:v16.2.0 shell -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring --fsid d0ab36da-4449-11ed-8433-001a4aab830c -e sha1=b1d2be3a2d73e5f2bd95bd349cfe2d8f9d4bca75 -- bash -c 'radosgw-admin zonegroup create --rgw-zonegroup=default --master --default'"
```
I think the now uncommented rgw realm in the start-distro files is creating the same thing as the rgw realm/zonegroup/zone create commands in 3-upgrade files. Going to just remove the rgw realm from the 1-distro files.

File: qa/suites/orch/cephadm/mgr-nfs-upgrade/1-bootstrap/octopus.yaml (hunk 1 above)
Comment (adk3798) [reply qa/suites/orch/cephadm/mgr-nfs-upgrade/1-bootstrap/octopus.yaml:4]: added a 16.2.0.yaml here

**Comment by adk3798** [comment]
https://pulpito.ceph.com/adking-2022-10-05_13:02:39-orch:cephadm-wip-adk-testing-2022-10-05-0715-distro-default-smithi/

2 failures:

- 1 seemingly infra thing installing an selinux package https://tracker.ceph.com/issues/57771
- Failure in initial part of staggered upgrade while manually redeploying the mgr daemons with new images. This will need more investigation but I don't think any of the PRs in the run were necessarily causing the issue.

**Pull request merged**